src/components/member-page/donation/table/row-detail.js

```diff
@@ -0,0 +1,317 @@
+              <P1Gray800 text={receiptAddress} />
+            </div>
+            <div className="row">
+              <P2Gray600 text="其他" />
+              <P1Gray800 text="我願意將全名公開在《報導者》的捐款徵信名冊" />
+            </div>
+          </MobileOnly>
+        </ReceiptInfoBox>
+        <PaymentInfoBox>
+          <P1Gray800 weight={P1.Weight.BOLD} text="扣款方式" />
+          <Divider />
+          <div className="card-number">
+            <P1Gray800 text={CardTypeDictionary[cardType]} />
+            <DotGroups />
+            <P1Gray800 text={cardLastFour} />
+          </div>
+        </PaymentInfoBox>
+        <PaymentRecordBox>
+          <P1Gray800 weight={P1.Weight.BOLD} text="扣款紀錄" />
+          <Divider />
+          <TabletAndAbove>
+            <div className="payment-record-table">
+              {periodicHistory.map((history, idx) => {
+                return (
+                  <React.Fragment key={idx}>
+                    <P1Gray800
+                      className="payment-date"
+                      text={formattedDate(new Date(history.created_at))}
+                    />
+                    <P1Gray800
+                      className="payment-number"
+                      text={history.order_number}
+                    />
+                    <P1Gray800
+                      className="payment-amount"
+                      text={history.amount}
+                    />
+                    <div className="payment-status">
+                      <StatusBadge
+                        status={history.status}
+                        type={DonationType.PERIODIC}
+                      />
+                    </div>
+                  </React.Fragment>
+                )
+              })}
+            </div>
+          </TabletAndAbove>
+          <MobileOnly>
+            <div className="payment-record-table">
+              {periodicHistory.map((history, idx) => {
+                return (
+                  <div className="grid-row" key={idx}>
+                    <P1Gray800
+                      className="payment-date"
+                      text={formattedDate(new Date(history.created_at))}
+                    />
+                    <P1Gray800
+                      className="payment-amount"
+                      text={history.amount}
+                    />
+                    <div className="payment-status">
+                      <StatusBadge
+                        status={history.status}
+                        type={DonationType.PERIODIC}
+                      />
+                    </div>
+                    <P1Gray800
+                      className="payment-number"
+                      text={history.order_number}
+                    />
+                  </div>
+                )
+              })}
+            </div>
+          </MobileOnly>
+        </PaymentRecordBox>
+      </LoadingMask>
+    </RowDetail>
+  )
+})
+
+TableRowDetail.propTypes = {
+  record: PropTypes.object.isRequired,
```

Comment:
Could we specify the shape of `record` object here

Reply:
It will be implemented after the spec is confirmed.